src/main/java/main/EmployeesProcessor.java

```diff
@@ -0,0 +1,388 @@
+                }
+                break;
+            }
+            case STREAM:{
+                lastNameOfEmployeesWithConcatenation = employeeList.stream()
+                        .filter(s->s.getAge()>70)
+                        .map(s->s.getLastName().concat(concatedStringToLastName))
+                        .collect(Collectors.toList());
+            }
+        }
+        return lastNameOfEmployeesWithConcatenation;
+    }
+
+    /**
+     * TASK #7
+     * Знайти всіх працівників, хто живе у києві та повернути колекцію унікальних їх імен
+     */
+    public Set<String> getDistinctEmployeesNameByAddress(List<Employee> employeeList, ExecutionType executionType, String address){
+        Set<String> distinctEmployeesName = new HashSet<>();
+        switch (executionType){
+            case FOREACH:{
+                for (Employee e:employeeList){
+                    if (e.getAddress().equals(address)){
+                        distinctEmployeesName.add(e.getFirstName());
+                    }
+                }
+                break;
+            }
+            case ITERATOR:{
+                Iterator<Employee> employeeIterator = employeeList.iterator();
+                while (employeeIterator.hasNext()){
+                    Employee employee = employeeIterator.next();
+                    if (employee.getAddress().equals(address)){
+                        distinctEmployeesName.add(employee.getFirstName());
+                    }
+                }
+                break;
+            }
+            case STREAM:{
+                distinctEmployeesName = employeeList.stream()
+                        .filter(s->s.getAddress().equals(address))
+                        .map(s->s.getFirstName())
+                        .collect(Collectors.toSet());
+                break;
+            }
+        }
+        return distinctEmployeesName;
+    }
+
+    /**
+     * TASK #8
+     * Вивести дані просто кожного працівника в консоль і витягнути колекцію адрес
+     */
+    public Set<String> getToStringAndAddressOfEmployees(List<Employee> employeeList, ExecutionType executionType){
+        Set<String> addressOfEmployees = new HashSet<>();
+        switch (executionType){
+            case FOREACH:{
+                employeeList.forEach(s->System.out.println(s.toString()));
+                for (Employee e:employeeList){
+                    addressOfEmployees.add(e.getAddress());
+                }
+                break;
+            }
+            case ITERATOR:{
+                employeeList.forEach(s->System.out.println(s.toString()));
+                Iterator<Employee> employeeIterator = employeeList.iterator();
+                while (employeeIterator.hasNext()){
+                    addressOfEmployees.add(employeeIterator.next().getAddress());
+                }
+                break;
+            }
+            case STREAM:{
+               employeeList
+                       .forEach(System.out::println);
```

Comment:
I was asking to complete this using 1 line, try to use peek()

Reply:
>  case STREAM:{
                addressOfEmployees = employeeList.stream()
                        .peek(System.out::println)
                        .map(Employee::getAddress)
                        .collect(Collectors.toSet());

>